DigitalLearningSolutions.Web/Styles/trackingSystem/allDelegates.scss

```diff
@@ -0,0 +1,29 @@
+﻿@import "~nhsuk-frontend/packages/core/all";

```

Comment:
Could we use the NHS breakpoints here or are they not working well for our use case?

Reply:
Not working well, eg. the largest nhs size is something like 960px, but we get unsightly wrapping from 1024px so need a breakpoint there.

Reply:
It's the same problem as I said for the pagination. A 3/4 column is effectively a tablet below 1024px (3/4 is 768px) but for some reason NHS have decided large-desktop is 990px even though that's actually quite small...

Reply:
Ok - I'll add a note on the all delegates search/sort ticket to refactor this as a separate MR. 👍